src/main/resources/api/service_openapi.yaml

```diff
@@ -459,6 +459,101 @@ components:
         hybridConnectionName:
           type: string
           description: The user-provided name to identify the Relay HybridConnection.
+    AzureContainerInstance:
```

Comment:
I think for `CloudResourceUid` in Janitor, we only need to capture enough information to be able to delete the object. In most cases that is (name, resourceGroup). Wondering if extraneous properties can be removed here?